ci/deploy.yaml

```
@@ -2,15 +2,17 @@ platform: linux
 image_resource:
   type: docker-image
   source:
-    repository: ((image_registry))/eq-app-deploy-image
-    tag: ((deploy_image_version))
+    repository: gcr.io/google.com/cloudsdktool/cloud-sdk
+    tag: latest
```

The full image is not needed. The `slim` version should be used.
```suggestion
    tag: slim
```